alf/drivers/threads.py

```diff
@@ -117,7 +117,15 @@ def repeat_shape_n(nested_spec, n):
         lambda t: tf.TensorSpec([n] + list(t.shape), t.dtype), nested_spec)
 
 
+LearningBatch = namedtuple("LearningBatch", [
+    "time_step", "state", "policy_step", "act_dist_param", "next_time_step",
+    "env_id"
```

Comment:
actually env_id never been used, and is value was set to  no meaning  `tf.ones()`